lib/praxis-blueprints/blueprint.rb

```
@@ -114,20 +123,15 @@ def self.domain_model(klass = nil)
     end
 
     def self.check_option!(name, value)
-      case name
-      when :identity
-        raise Attributor::AttributorException, "Invalid identity type #{value.inspect}" unless value.is_a?(::Symbol)
-        return :ok
-      else
-        return Attributor::Struct.check_option!(name, value)
-      end
+      Attributor::Struct.check_option!(name, value)
     end
 
     def self.load(value, context = Attributor::DEFAULT_ROOT_CONTEXT, **options)
       case value
       when self
         value
       when nil, Hash, String
+        # TODO: huh? what if loading fails...shouldn't we capture errors? probably no, in validate?
```

Loading can't silently fail, it either works or throws an exception.